editor/editor_inspector.cpp

```diff
@@ -3271,16 +3294,6 @@ void EditorInspector::update_tree() {
 			continue;
 		}
 
-		// Checkable and checked properties.
-		bool checkable = false;
-		bool checked = false;
-		if (p.usage & PROPERTY_USAGE_CHECKABLE) {
-			checkable = true;
-			checked = p.usage & PROPERTY_USAGE_CHECKED;
-		}
-
-		bool property_read_only = (p.usage & PROPERTY_USAGE_READ_ONLY) || read_only;
-
```

Comment:
Why was this moved?

Reply:
I'd say this PR should be only style changes and any functional changes should be separate, style improvements and code changes shouldn't be mixed, even if this shouldn't affect anything